src/bootlint.js

```diff
@@ -162,6 +187,68 @@ var LocationIndex = _location.LocationIndex;
         return runs;
     }
 
+    /**
+     * This function returns the browser window object, or null if this is not running in a browser environment.
+     * @returns {window object or null}
+     */
+    function getBrowserWindowObject() {
+        var theWindow = null;
+        try {
+            /*eslint-disable no-undef, block-scoped-var */
+            theWindow = window;// jshint ignore:line
+            /*eslint-enable no-undef, block-scoped-var */
+        }
+        catch (e) {
+            // deliberately do nothing
+        }
+
+        return theWindow;
+    }
+
+    function findSemverMatches(pathSegments) {
+        /* @covignore */
```

Comment:
I believe I've removed the one you referenced and just pushed it. I apologize if I removed the wrong thing though